sh/lint.sh

```
@@ -1,19 +1,24 @@
-python3 -m ruff check phlop tests
-for FILE in ${PY_FILES[@]}; do
+  set -e
 
-  autoflake -i "$FILE"
+  PY_FILES=$(find . -name "*.py")
 
+  python3 -m black phlop tests
+  pylint --errors-only phlop tests
+  # isort phlop tests
+  python3 -m ruff check phlop tests
+  for FILE in ${PY_FILES[@]}; do
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)